src/test/java/com/wikia/webdriver/testcases/mercurytests/SEOTests.java

```diff
@@ -142,4 +152,54 @@ public void MercurySEOTest_001_MetaTags_CanonicalLink() {
         seo.isOgTypeArticle()
     );*/
   }
+
+  @Test(groups = {"MercurySEOTest_002", "MercurySEOTests", "Mercury"})
+  public void MercurySEOTest_002_MetaTags_Robots_SectionToMainPage() {
+    SEOUtils seoUtils = new SEOUtils(driver);
+    wikiURL = urlBuilder.getUrlForWiki(MercuryWikis.MERCURY_CC);
+    CuratedContentPageObject section = new CuratedContentPageObject(driver);
+    section.openCuratedContentPage(wikiURL, MercurySubpages.CC_SECTION_CATEGORIES);
+
+    Assertion.assertTrue(seoUtils.isRobots(ROBOTS_TAG_ATTRIBUTES_NOINDEX));
```

Comment:
you changed the method name here: https://github.com/Wikia/selenium-tests/pull/878/files#diff-c8820aadc641dc8ecd6d491082c85247R126 but not in the line above.

Does it work?